packages/nvidia_nat_langchain/src/nat/plugins/langchain/eval/trajectory_evaluator.py

```
@@ -72,24 +148,67 @@ def __init__(self, llm: BaseChatModel, tools: list[BaseTool] | None = None, max_
+            if isinstance(e, ValueError) and "not enough values to unpack" in str(e):
+                logger.warning("Trajectory judge output parsing failed [lane=%s item_id=%s question=%r]: %s",
+                               lane,
+                               item_id,
+                               question_text,
+                               e)
+            else:
+                logger.exception("Error evaluating trajectory [lane=%s item_id=%s question=%r]: %s",
+                                 lane,
+                                 item_id,
+                                 question_text,
+                                 e)
```

Ruff (0.15.4)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)